eclair-core/src/test/scala/fr/acinq/eclair/channel/states/ChannelStateTestsHelperMethods.scala

```diff
@@ -496,6 +499,8 @@ trait ChannelStateTestsBase extends Assertions with Eventually {
     // check that we store the local txs without sigs
     localCommit.commitTxAndRemoteSig.commitTx.tx.txIn.foreach(txIn => assert(txIn.witness.isNull))
     localCommit.htlcTxsAndRemoteSigs.foreach(_.htlcTx.tx.txIn.foreach(txIn => assert(txIn.witness.isNull)))
+    val probe = TestProbe()
+    systemA.eventStream.subscribe(probe.ref, classOf[OnchainPubkeyRefresher.Command])
```

Comment:
Unused (+ import)

Reply:
done in https://github.com/ACINQ/eclair/pull/2565/commits/e50a30eb4fbfb35dd74fd521bd2d9b1255315207